doc/nrf/releases_and_maturity/migration/migration_guide_3.0.rst

```diff
@@ -333,6 +333,17 @@ Download client
 
            err = downloader_deinit(&dl);
 
+Modem SLM
+---------
+
+.. toggle::
+
+   For applications and samples using the :ref:`lib_modem_slm` library:
+
+    * Replace the use of the ``CONFIG_MODEM_SLM_WAKEUP_PIN`` Kconfig option with the :kconfig:option:`CONFIG_MODEM_SLM_POWER_PIN`` Kconfig option.
+    * Replace the use of the ``CONFIG_MODEM_SLM_WAKEUP_TIME`` Kconfig option with the :kconfig:option:`CONFIG_MODEM_SLM_POWER_PIN_TIME` Kconfig option.
+    * Replace the use of the :c:func:`modem_slm_wake_up` function with the :c:func:`modem_slm_power_pin_toggle` function.
```

Comment:
```suggestion
    * Replace the ``CONFIG_MODEM_SLM_WAKEUP_PIN`` Kconfig option with :kconfig:option:`CONFIG_MODEM_SLM_POWER_PIN``.
    * Replace the ``CONFIG_MODEM_SLM_WAKEUP_TIME`` Kconfig option with :kconfig:option:`CONFIG_MODEM_SLM_POWER_PIN_TIME`.
    * Replace the :c:func:`modem_slm_wake_up` function with :c:func:`modem_slm_power_pin_toggle`.
```

Reply:
This is what I had initially but changed it to be aligned with a previous migration guide 😄 I'll change it back.